pkg/agent/informant.go

```diff
@@ -194,7 +194,11 @@ func NewInformantServer(
 				logFunc = runner.logger.Infof
 			}
 
-			logFunc("Informant server exiting with retry: %v, err: %s", status.RetryShouldFix, status.Err)
+			errDesc := "nil"
+			if status.Err != nil {
+				errDesc = fmt.Sprintf("%s", status.Err)
+			}
+			logFunc("Informant server exiting with: retry=%v, err=%s", status.RetryShouldFix, errDesc)
```

Comment:
I'd just use %q and let the empty quote marks to give an indication that there's an issue 

I might also add a method to status that's like `ReportError` just for clarity.

Reply:
double-checking I understand - you're suggesting that if `err == nil`, we should print `err=""`?

If so, I'd rather the logs be more explicit about that. Although on a related note, I think it'd make sense to set `errDesc = fmt.Sprintf("%q", status.Err.Error())` instead, so that it's unambiguous.

Reply:
I mean I don't think it super matters. I think this is a lot of code to get, essentially what you get from `fmt.Sprintf("'%v', err)`

https://go.dev/play/p/dp58wuSUGD5

Reply:
Ended up going with just using `%v` for the error